tooling/api/src/path.ts

```diff
@@ -640,12 +639,10 @@ async function extname(path: string): Promise<string> {
  * Returns the last portion of a `path`. Trailing directory separators are ignored.
  * @example
  * ```typescript
- * import { basename, resolveResource } from '@tauri-apps/api/path';
- * const resourcePath = await resolveResource('app.conf');
- * const base = await basename(resourcePath);
+ * import { basename } from '@tauri-apps/api/path';
+ * const base = await basename('path/to/app.conf');
```

Comment:
```suggestion
 * const base = await basename('path/to/app.conf');
 * assert(base === 'app.conf');
```

Reply:
this actually added another assert statement